frontend/src/hooks/api/getters/useConstraintsValidation/useConstraintsValidation.ts

```diff
@@ -16,25 +24,29 @@ export const useConstraintsValidation = (
             return;
         }
 
-        const validationRequests = constraints
-            .filter(constraint => {
-                const hasValues =
-                    Array.isArray(constraint.values) &&
-                    constraint.values.length > 0;
-                const hasValue = Boolean(constraint.value);
+        const invalidConstraints = constraints.find(item => !isValid(item));
+        if (invalidConstraints) {
+            setValid(false);
+            return;
+        }
 
-                return hasValues || hasValue;
```

Comment:
this is wrong. we were excluding locally invalid constraints from the remote check without saying those items are invalid